post.md

```diff
@@ -0,0 +1,226 @@
+# Robot Framework - using Request library for control RPI GPIO's
+
+Requests library is one of the most popular libraries implemented for Robot
+Framework. It is very important for testing scattered applications, so this is
+the first library I got to know in the Robot Framework. The library requires 
+installing `robotframework-requests` and `requests` for proper working. 
+`Collections` library is a standard built-in library.
```

Comment:
This explanation of `Collections` library is a little bit out of context. @konopskyyyy can You link it somehow to previous sentences? Or maybe move to other section if possible?